battlegear mod src/minecraft/mods/battlegear2/coremod/BattlegearLoadingPlugin.java

```diff
@@ -10,6 +11,7 @@
 
 @TransformerExclusions({"mods.battlegear2.coremod"})
 @Name("Mine and Blade: Battlegear2")
+@SortingIndex(1500)
```

Comment:
While i understand FMLDeobfTweaker is put at index 1000 and any above FMLPluginWrapper for IFMLLoadingPlugin would be sorted after, is this 1500 value specific for "Minecraft Forkage" support ?